codespell_lib/data/dictionary.txt

```diff
@@ -25050,14 +25169,21 @@ framwework->framework
 framwork->framework
 framworks->frameworks
 framws->frames
-frane->frame
+frane->frame, France, crane, franc,
+franes->frames, cranes, francs,
+franework->framework
+franeworks->frameworks
 frankin->franklin
 Fransiscan->Franciscan
 Fransiscans->Franciscans
 franzise->franchise
 fration->fraction, ration,
 frational->fractional, rational,
 frations->fractions, rations,
+freame->frame
```

Comment:
Is `freame` ever a typo of `cream`? In any case, what about `freamed->creamed`?

Reply:
I couldn't find both of these.